core/data/clients/InfluxEvent.go

```diff
@@ -0,0 +1,169 @@
+/*******************************************************************************
+ * Copyright 2017 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
```

Comment:
I would suggest using SPDX headers (https://spdx.org/) for simplicity - something like this: https://github.com/edgexfoundry/edgex-go/blob/master/export/filter.go#L1-L5